crates/libeq_wld/src/lib.rs

```diff
@@ -349,12 +349,8 @@ impl<'a> Material<'a> {
             })
     }
 
-    pub fn transparency_flags(&self) -> &TransparencyFlags {
-        &self.fragment.transparency_flags
-    }
-
-    pub fn mask_color_coord(&self) -> &(f32, f32) {
-        &self.fragment.mask_color_coord
```

Comment:
I suspect that these fields _may_ not be the mask color 😭. But you may have more information. (more comments below)